components/provisioner/internal/api/mockresolver.go

```diff
@@ -0,0 +1,196 @@
+package api
+
+import (
+	"context"
+	"sort"
+	"time"
+
+	"github.com/kyma-incubator/compass/components/provisioner/pkg/gqlschema"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/uuid"
+)
+
+type MockResolver struct {
+	repository map[string]RuntimeOperation
+}
+
+type externalMutationResolver struct {
```

Comment:
Why we need externalMutationResolver and externalQueryResolver types?In Mutation() and Query() we can directly use MockResolver type.